lib/document_view.dart

```diff
@@ -45,12 +42,31 @@ class DocumentViewController {
 
   final MethodChannel _channel;
 
-  Future<void> openDocument(String document,
-      {String password, Config config}) {
+  Future<void> openDocument(String document, {String password, Config config}) {
     return _channel.invokeMethod('openDocument', <String, dynamic>{
       'document': document,
       'password': password,
       'config': jsonEncode(config)
     });
   }
-}
\ No newline at end of file
+
+  Future<void> importAnnotationCommand(String xfdfCommand) {
+    return _channel.invokeMethod('importAnnotationCommand',
+        <String, dynamic>{'xfdfCommand': xfdfCommand});
+  }
+
+  Future<void> importBookmarkJson(String bookmarkJson) {
+    return _channel.invokeMethod(
+        'importBookmarkJson', <String, dynamic>{'bookmarkJson': bookmarkJson});
+  }
+
+  Future<String> saveDocument() async {
+    return _channel.invokeMethod('saveDocument');
+  }
+
+  Future<dynamic> getPageCropBox(int pageNumber) {
+    return _channel.invokeMethod('getPageCropBox', <String, dynamic>{
```

Comment:
Can these strings also be declared as constants in Dart? 

Looks like the syntax may be `const String GET_PAGE_CROP_BOX = "getPageCropBox";`.

I'm not sure how to idiomatically name a Dart const string, if all caps is appropriate or not.